elements/pl_number_input/pl_number_input.py

```diff
@@ -79,7 +79,7 @@ def render(element_html, element_index, data):
         with open('pl_number_input.mustache', 'r') as f:
             html = chevron.render(f, html_params).strip()
     elif data['panel'] == 'answer':
-        a_tru = data['correct_answers'].get(name, None)
+        a_tru = pl.from_json(data['correct_answers'].get(name, None))
```

Comment:
Am I correct in thinking that `pl.from_json()` here doesn't actually do anything? Would it be cleaner to leave it off, or are we wanting it for safety or consistency?

Same comment applies to the `pl.from_json()` calls later in this file.

Reply:
You are correct that `pl.from_json()` and `pl.to_json()` do nothing to floats, so are not necessary in `pl_number_input`.

You'll recall I had looked at a couple of python libraries that do a more complete job of JSON encoding and decoding, and in particular that put *everything* into some kind of wrapper dictionary (including floats). We had decided not to use any of these libraries. If we used them - or built something like them ourselves - at some point in the future, and if we hadn't already been consistent in our use of `pl.from_json()`, then we would have to go back and add these calls in elements that involved floats and other data structures. So the thought was consistency, yes, and future compatibility.

That being said, if we were really heading in that direction, then we would want to use `pl.to_json()` calls in these input elements to encode `data['submitted_answers']` - we don't do that right now.

So maybe the best thing to do is remove `pl.from_json()` calls in `pl_number_input`, as you suggest. Any other thoughts?

Reply:
I agree that at this point it's more explicit to only use the calls where needed. Makes it easier to explain what we are doing. If we change later then we can just go through and change everything :-)

Reply:
ok, done.